src/main/java/team/gif/robot/RobotMap.java

```diff
@@ -40,4 +40,8 @@ public abstract class RobotMap {
     public static final int SOLENOID_COLLECTOR_REVERSE = 0;
     public static final int COLLECTOR_LEFT_MOTOR = 0;
     public static final int COLLECTOR_RIGHT_MOTOR = 0;
+
+    //Led
+    public static final int LED_LENGTH = 32;
```

Comment:
LED_LENGTH should be declared in Constants.java
and I suggest you change it to NUM_LEDS